docker/fedora37

```diff
@@ -3,16 +3,18 @@ FROM fedora:37
 # note: at variance with centos7, here we have to explicitly install gcc
 # mdtraj 1.9.7 does not compile with python 3.11 unless installed from source
 # see https://github.com/mdtraj/mdtraj/issues/1761
-RUN yum -y update \
- && yum -y group install "Development Tools" \
- && yum -y install gcc-c++ \
- && yum -y install environment-modules gawk vim wget \
-                   lapack-devel blas-devel zlib-devel gsl-devel fftw-devel openmpi-devel boost-devel \
-                   python3 python3-devel python3-pip \
- && pip3 install numpy \
- && pip3 install cython numpy pandas \
- && pip3 install MDAnalysis \
- && pip3 install git+https://github.com/mdtraj/mdtraj@1.9.7
+RUN yum -y update 
+RUN yum -y group install "Development Tools" 
+RUN yum -y install gcc-c++ 
+RUN yum -y install environment-modules gawk vim wget lapack-devel blas-devel \
+                   zlib-devel gsl-devel fftw-devel openmpi-devel boost-devel \
+                   python39 python-devel python-pip 
```

Comment:
@Iximiel penso il problema sia qui, devi cambiare
`pyhon39 python-devel python-pip `
in
`pyhon39 python39-devel python39-pip`


vedi [questo link](https://access.redhat.com/documentation/it-it/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_installing-and-using-python_configuring-basic-system-settings)

Reply:
I have already [tried here](https://github.com/plumed/plumed2/actions/runs/5740695031/job/15559106718)
but yum returns

>No match for argument: python39-devel
>No match for argument: python39-pip

So I opted for `ensurepip`, and it seems to work until mdtraj